### PR TITLE
Proposing Governance

### PR DESCRIPTION
**Pull request by rattermeyer**
Proposed a governance document based on articles from oss-watch

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+* providing moral support (a ‘thank you’ goes a long way)
+* providing financial support (the software is open source, but its developers need to eat)
+
+Users who continue to engage with the project and its community will often become more and more
+involved. Such users may find themselves becoming contributors, as described in the next section. 
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become
+a contributor, and contributions can take many forms, as detailed in a separate document.
```
Comment (michaelsauter) [inline governance.md]: This mentions a separate document but the PR doesn't have another doc?

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+It is important to recognise that commitership is a privilege, not a right. That privilege must be
+earned and once earned it can be removed by the PMC (see next section) in extreme circumstances.
+However, under normal circumstances committership exists for as long as the committer wishes to
+continue engaging with the project.
+
+A committer who shows an above-average level of contribution to the project, particularly with
+respect to its strategic direction and long-term health, may be nominated to become a member of the
+PMC. This role is described below. 
+
+### Project Management Committee
```
Comment (gerardcl) [inline governance.md:94]: This first paragraph is repeated below once PMC chapter starts again

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+Committership allows contributors to more easily carry on with their project related activities by
+giving them direct access to the project’s resources. 
+That is, they can make changes directly to project outputs, without having to submit changes via patches.
+
+This does not mean that a committer is free to do what they want. In fact, committers have no more
+authority over the project than contributors.  While committership indicates a valued member of the
+community who has demonstrated a healthy respect for the project’s aims and objectives, their work
+continues to be reviewed by the community before acceptance in an official release. The key
+difference between a committer and a contributor is when this approval is sought from the community.
+A committer seeks approval after the contribution is made, rather than before.
```
Comment (michaelsauter) [inline governance.md]: Hmm. I think a committer's "default mode" should still be to seek approval before making contributions. 

IMHO, at least the workflow should be the same: a committer should still open PRs and be open to feedback. The difference being that you can overrule and still merge even if there are concerns from the community.

Comment (rattermeyer) [reply governance.md]: I was thinking about this as well, agree with you. It should always be reviewed by another person (4-eyes-principle)

**Review by gerardcl — CHANGES_REQUESTED** [review]
see comment at Project Management Committee chapter

**Pull request merged**